### ISSUE TITLE
How to use in plain english

### ISSUE THREAD

**Issue by Globik** [llamerada-jp/webrtc-cpp-sample#1]
Ecscuse me, I have compiled your sample on windows 7 32bit machiene. It works. But should I run two programms, two console shells? And when and where should I run the command "send"?

**Comment by Globik** [comment]
What is the order? First shell=> ice1  sdp1? Second shell=> ice2 sdp2? And what is "; " meaning?

**Comment by llamerada-jp** [comment]
I add README.en.md a little while ago. It is english translation of README.md. Would you read that document. And please tell me some questions after retry.

**Comment by Globik** [comment]
@llamerada-jp  Yes! I got it! All steps one by one are successfully. And also the command "send". I see, I see now the meaning of " ; " mark. One test-programm for two consoles. It is a hardwork with copypaste.  But worth it. Anyway thank you very much for your modern datachannel sample and the instruction in english. And now I see that webrtc-native can work also on windows 7 32-bit. Cheers!

**Comment by llamerada-jp** [comment]
Your welcome. ㄟ(ﾟ▽ﾟ*)ㄏ
